Polyhedron/demo/Polyhedron/Plugins/Point_set/Surface_reconstruction_plugin.cpp

```diff
@@ -889,13 +984,26 @@ void Polyhedron_demo_surface_reconstruction_plugin::automatic_reconstruction
 	      std::cerr << "Advancing front reconstruction... ";
 	      time.restart();
 
-	      Scene_polyhedron_item* reco_item = new Scene_polyhedron_item(Polyhedron());
-	      SurfaceReconstruction::advancing_front (*points, reco_item, 10. * (std::max)(noise_size, aniso_size));
-	      
-	      reco_item->setName(tr("%1 (advancing front)").arg(scene->item(index)->name()));
-	      reco_item->setColor(Qt::lightGray);
-	      reco_item->setRenderingMode(FlatPlusEdges);
-	      scene->addItem(reco_item);
+              if(mw->property("is_polyhedron_mode").toBool())
```

Comment:
Where does this come from? I did not manage to find something that looked like a "_polyhedron mode_" in the menus, but that might be something else. On my tests, it seems that it is always a _Surface Mesh_ that is produced (which is fine, but I wonder how to switch to _Polyhedron_, as it seems possible from what I read in the code).